app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def create
+    @review = Review.find(params[:review_id])
+    @comment = Comment.new(comment_params)
+    @comment.user = current_user
+    @comment.review = @review
+
+    if @comment.save
+      redirect_to review_path(@review),
+        notice: "Comment has been posted sucessfully"
+    else
+      render 'reviews/show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.